velox/type/Type.h

```diff
@@ -530,7 +531,7 @@ class Type : public Tree<const TypePtr>, public velox::ISerializable {
   VELOX_FLUENT_CAST(Hugeint, HUGEINT)
   VELOX_FLUENT_CAST(Real, REAL)
   VELOX_FLUENT_CAST(Double, DOUBLE)
-  VELOX_FLUENT_CAST(Varchar, VARCHAR)
+
   VELOX_FLUENT_CAST(Varbinary, VARBINARY)
```

Comment:
Since Varchar and Varbinary types tend to go together, should we also support VarbinaryN?

Reply:
The reason I had not considered VarbinaryN is because Presto does not support it. 
```
presto:default> create table varbin (col1 varbinary(10));
Query 20240823_220701_00004_uxz37 failed: line 1:22: Unknown type 'varbinary(10)' for column 'col1'
```
Of course, it doesn't mean that Velox cannot support it as a built-in type.
We can add it with the same design that we use with this PR.